support/mock/mockrt.go

```diff
@@ -803,6 +803,12 @@ type ReturnWrapper struct {
 
 func (r ReturnWrapper) Into(o runtime.CBORUnmarshaler) error {
 	b := bytes.Buffer{}
+	if r.V == nil {
```

Comment:
When applying a message with no return value (e.g. transfer of value) the return value is an empty byte array. This change allows `Into` to be called on a `ReturnWrapper` that has an empty value, previously it would panic. This is inline with the lotus implementation which handles empty returns without failing: https://github.com/filecoin-project/lotus/blob/9b867a221a2babcfc97a43b4b9f24054976ef975/chain/vm/runtime.go#L338-L344

Reply:
To fix this correctly we'll have to change all the call sites of ExpectSend (an example: https://github.com/filecoin-project/specs-actors/blob/master/actors/builtin/miner/miner_test.go#L435) since we pass `nil` for the return value. I'll make that change in a seperate PR and rebase onto it.

Reply:
Rather than changing the call sites, change ExpectSend to adapt nil to `EmptyValue` (which is what the real vm's (should) do).

Reply:
done